assets/src/components/Sidebar/CampaignSidebar.js

```diff
@@ -0,0 +1,220 @@
+import { PluginSidebar, PluginSidebarMoreMenuItem } from "@wordpress/edit-post";
```

Comment:
Hey @Inwerpsel this "PluginSidebarMoreMenuItem" is no longer used, BUT, I missed your comment about this, I guess you may think about re-adding it just in case, since the CSS fix for "Unpin from toolbar" could get translated if you set the WP admin to other languages, also it might be a bit too generic (what if other buttons on other toolbars have that label?). I think you can keep both fixes, adding a bit more scope to the CSS thing.

Reply:
Re-added it :+1: